google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddAdCustomizer.java

```diff
@@ -0,0 +1,498 @@
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v2.common.ExpandedTextAdInfo;
+import com.google.ads.googleads.v2.enums.AdCustomizerPlaceholderFieldEnum.AdCustomizerPlaceholderField;
+import com.google.ads.googleads.v2.enums.FeedAttributeTypeEnum.FeedAttributeType;
+import com.google.ads.googleads.v2.enums.PlaceholderTypeEnum.PlaceholderType;
+import com.google.ads.googleads.v2.errors.GoogleAdsError;
+import com.google.ads.googleads.v2.errors.GoogleAdsException;
+import com.google.ads.googleads.v2.resources.Ad;
+import com.google.ads.googleads.v2.resources.AdGroupAd;
+import com.google.ads.googleads.v2.resources.AttributeFieldMapping;
+import com.google.ads.googleads.v2.resources.Feed;
+import com.google.ads.googleads.v2.resources.FeedAttribute;
+import com.google.ads.googleads.v2.resources.FeedItem;
+import com.google.ads.googleads.v2.resources.FeedItemAttributeValue;
+import com.google.ads.googleads.v2.resources.FeedItemTarget;
+import com.google.ads.googleads.v2.resources.FeedMapping;
+import com.google.ads.googleads.v2.services.AdGroupAdOperation;
+import com.google.ads.googleads.v2.services.AdGroupAdServiceClient;
+import com.google.ads.googleads.v2.services.FeedItemOperation;
+import com.google.ads.googleads.v2.services.FeedItemServiceClient;
+import com.google.ads.googleads.v2.services.FeedItemTargetOperation;
+import com.google.ads.googleads.v2.services.FeedItemTargetServiceClient;
+import com.google.ads.googleads.v2.services.FeedMappingOperation;
+import com.google.ads.googleads.v2.services.FeedMappingServiceClient;
+import com.google.ads.googleads.v2.services.FeedOperation;
+import com.google.ads.googleads.v2.services.FeedServiceClient;
+import com.google.ads.googleads.v2.services.GoogleAdsServiceClient;
+import com.google.ads.googleads.v2.services.GoogleAdsServiceClient.SearchPagedResponse;
+import com.google.ads.googleads.v2.services.MutateAdGroupAdResult;
+import com.google.ads.googleads.v2.services.MutateAdGroupAdsResponse;
+import com.google.ads.googleads.v2.services.MutateFeedItemResult;
+import com.google.ads.googleads.v2.services.MutateFeedItemTargetsResponse;
+import com.google.ads.googleads.v2.services.MutateFeedItemsResponse;
+import com.google.ads.googleads.v2.services.MutateFeedMappingsResponse;
+import com.google.ads.googleads.v2.services.MutateFeedsResponse;
+import com.google.ads.googleads.v2.services.SearchGoogleAdsRequest;
+import com.google.ads.googleads.v2.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.joda.time.DateTime;
+
+/** Adds an ad customizer feed and associates it with the customer. Then it adds an ad
```

Comment:
Knit: put the `/**` and `*/` on separate lines than the comment itself, and the comment on lines in between to be consistent with other examples.

Reply:
Fixed by running google-java-format.